go/vt/vtadmin/api.go

```diff
@@ -87,6 +88,7 @@ func NewAPI(clusters []*cluster.Cluster, opts grpcserver.Options, httpOpts vtadm
 	router.HandleFunc("/clusters", httpAPI.Adapt(vtadminhttp.GetClusters)).Name("API.GetClusters")
 	router.HandleFunc("/gates", httpAPI.Adapt(vtadminhttp.GetGates)).Name("API.GetGates")
 	router.HandleFunc("/keyspaces", httpAPI.Adapt(vtadminhttp.GetKeyspaces)).Name("API.GetKeyspaces")
+	router.HandleFunc("/schema/{cluster_id}/{keyspace}/{table}", httpAPI.Adapt(vtadminhttp.GetSchema)).Name("API.GetSchema")
```

Comment:
I realize you started the disambiguation branch anyway so I expect you already have some plan 😎

Reply:
I think, given that you said this route scheme looks prettier, that we should keep it as-is because:

1. we're still able to reuse the logic of `cluster.GetSchema` between the two endpoints, so we're not duplicating too much
2. having two distinct rpcs and http routes lets us move some of the conditional logic out of the actual function and into the route definition (i.e. `GetSchema` doesn't have to that `len(clusterIds)` is _exactly_ 1, because it's not possible, but if we combine it with the `FindSchema` endpoint then we'll end up introducing a bunch of "if this then do thingAForGetSchema else do thingBForFindSchema" branching)

Reply:
Yeah makes sense!

Reply:
The /schema endpoint does seem a bit non-std instead of url params or nested resource paths like /schema/cluster/{cluster_id}/keyspace/{keyspace}/table/{table}. But it is more succinct and we can revisit this once we have all our endpoints in place.